mobile/lib/features/trade/trade_screen.dart

```diff
@@ -15,10 +13,81 @@ class TradeScreen extends StatefulWidget {
 class _TradeScreenState extends State<TradeScreen> {
   @override
   Widget build(BuildContext context) {
+    var fabShape = RoundedRectangleBorder(borderRadius: BorderRadius.circular(10.0));
+    double fabWidth = 100.0;
+
     return Scaffold(
         body: ListView(
-      padding: const EdgeInsets.only(left: 25, right: 25),
-      children: const [Center(child: Text("Trade Screen"))],
-    ));
+          padding: const EdgeInsets.only(left: 25, right: 25),
+          children: const [Center(child: Text("Trade Screen"))],
+        ),
+        floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
+        floatingActionButton: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              SizedBox(
+                  width: fabWidth,
+                  child: FloatingActionButton.extended(
+                    heroTag: "btnBuy",
+                    onPressed: () {
+                      showModalBottomSheet<void>(
+                        useRootNavigator: true,
+                        backgroundColor: Colors.green.shade50,
```

Comment:
Shall we introduce constants so that we can change them all at once, e.g. `GoLongGreen` (or similar) to indicate it's a green used for everything related to go long/buy in the trade screen.

Reply:
I invested a timebox to understand how to properly configure such kind of values in the `ThemeData`, but the timebox expired. I think it might still be a good idea to invest a bit of time into making this better, but that probably goes to far for this PR. 

I create https://github.com/get10101/10101/issues/98 to tackle this.